sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/resource/_lambda.py

```diff
@@ -0,0 +1,63 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from os import environ
+
+from opentelemetry.sdk.resources import Resource, ResourceDetector
+from opentelemetry.semconv.resource import (
+    CloudPlatformValues,
+    CloudProviderValues,
+    ResourceAttributes,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AwsLambdaResourceDetector(ResourceDetector):
+    """Detects attribute values only available when the app is running on AWS
+    Lambda and returns them in a Resource.
+
+    Uses Lambda defined runtime enivronment variables. See more: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime
+    """
+
+    def __init__(self, raise_on_error=False):
+        super().__init__(raise_on_error=raise_on_error)
```

Comment:
Ah thank you for this catch, in fact we don't need this since the parent class does the exact same thing.

https://github.com/open-telemetry/opentelemetry-python/blob/bb41b81093d4010dadb233f146e8ba04973ec5a3/opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py#L255-L257

Will remove!